evm_tester/src/vm/zk_ee/mod.rs

```diff
@@ -133,6 +133,7 @@ impl ZkOS {
             gas_per_pubdata: Default::default(),
             block_number: system_context.block_number as u64,
             timestamp: system_context.block_timestamp as u64,
+            chain_id: 37,
```

Comment:
Maybe we should use value from context? Instead of hardcoding 37

Reply:
Yes ofc, pushed a new version, thanks